### PR TITLE
Portal 1436/dashboard status boxes

### PR DESCRIPTION
**Pull request by tbolt**
# Description
* Adds status boxes to dashboard

### CODE REVIEW

**Review by ZacharyRizer — APPROVED** [review]
Looks great!

File: src/screens/Dashboard.tsx [hunk 1]
```diff
@@ -33,27 +35,29 @@ function Dashboard() {
...
+            Take a tour
+          </Button>
+          <Button
+            className={styles["learn-more-btn"]}
+            ariaLabel="learn more"
+            variation="outline">
+            Learn more
+          </Button>
+        </div>
+        */}
```
Comment (sublettk) [inline src/screens/Dashboard.tsx:60]: Let's remove this commented code.

File: src/styles/PieChart.module.css [hunk 2]
```diff
@@ -1,6 +1,8 @@
 .pie-chart-container {
+  background-color: #fff;
   border: 0.1rem solid #e0e0e0;
```
Comment (sublettk) [inline src/styles/PieChart.module.css:3]: Are we able to use the component library's color tokens?

File: src/styles/StatusBoxes.module.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
+.status-boxes-container {
+  display: flex;
+  gap: 1.7rem;
+  margin-bottom: 1.7rem;
+}
+
+.status-boxes-box {
+  border: 0.1rem solid #e0e0e0;
```
Comment (sublettk) [inline src/styles/StatusBoxes.module.css:8]: Are we able to use the component library's color tokens?

**Review by sublettk — CHANGES_REQUESTED** [review]

File: src/styles/PieChart.module.css (hunk 2 above)
Comment (tbolt) [reply src/styles/PieChart.module.css:3]: We are not exporting pure white from `cdc-react`. For all the other colors I updated to use the color tokens (css vars)

**Review by sublettk — APPROVED** [review]
LGTM

**Pull request merged**